src/main/java/uk/gov/hmcts/probate/service/tasklist/TaskStateRenderer.java

```diff
@@ -159,28 +163,31 @@ private String renderSendDocsDetails(TaskState sendDocsState, String caseId, Cas
         return sendDocsState == TaskState.NOT_AVAILABLE ? "" :
                 DetailsComponentRenderer.renderByReplace(SEND_DOCS_DETAILS_TITLE,
                         SendDocumentsDetailsHtmlTemplate.DOC_DETAILS.replaceFirst("<refNum/>", caseId)
-                .replaceFirst(ORIGINAL_WILL, keyValues.getOrDefault("originalWill", ""))
-                .replaceFirst(IHT_TEXT, keyValues.getOrDefault("ihtText", ""))
-                .replaceFirst(IHT_FORM, keyValues.getOrDefault("ihtForm", ""))
-                .replaceFirst(RENOUNCING_EXECUTORS, keyValues.getOrDefault("renouncingExecutors", ""))
-                .replaceFirst(PA16_FORM, keyValues.getOrDefault("pa16Form", ""))
-                .replaceFirst(PA17_FORM, keyValues.getOrDefault("pa17Form", ""))
-                .replaceFirst(IHT_ESTATE_207, keyValues.getOrDefault("ihtEstate207", ""))
+                            .replaceFirst(ORIGINAL_WILL, keyValues.getOrDefault("originalWill", ""))
```

Comment:
can you adjust your formating to be the same as current please, so we dont have these big changes. And then apply to the whole of your changeset